.github/workflows/main.yml

```diff
@@ -143,3 +143,33 @@ jobs:
           tag: ${{ github.ref }}
           prerelease: ${{ !startsWith(github.ref, 'refs/tags/v') }}
           overwrite: true
+
+  arm-v6:
+    name: ARM v6 binary
```

Comment:
Just above we have:
```
run: docker run --rm dockcross/${{ matrix.arch_name }} > ./dockcross; chmod +x ./dockcross
```

And apparently it supports armv6:
https://github.com/dockcross/dockcross/tree/master/linux-armv6

Can we add it to the matrix of distros instead of duplicating this?

Reply:
@bbworld1 any idea?

Reply:
Oh, I was not rebased over the latest master, so I was not aware that armv7 builds are already available, going to close the PR.